### ISSUE TITLE
[Bug?] Issues with some models

### ISSUE THREAD

**Issue by brunoodle** [darktrovx/interact#17]
Hey all,

I recently updated to the newest version and now a few models don't seem to show the interact key. I can confirm `prop_fleeca_atm` and `prop_med_bag_01b` display this issue. The interact key appeared correctly before [this](https://github.com/darktrovx/interact/commit/768a6eb333c19c898fa4da7202523d05d57347a2) push, and anything past that seems to be broken for me. Here's a video and screenshot to demonstrate what I'm explaining: https://streamable.com/xgh3uy

![image](https://github.com/darktrovx/interact/assets/71146760/3ec73f00-3c90-4be9-9240-b168655967e8)

Not sure if it matters, but here's my options/exports. I've tried adjusting the distances, but nothing works.
```
local atmModels = {
`prop_atm_03`,
`prop_atm_02`,
`prop_atm_01`,
`prop_fleeca_atm`,

}

for i = 1, #atmModels do
    exports.interact:AddModelInteraction({
        model = atmModels[i],
        offset = vec3(0.0, 0.0, 1.0), -- optional
        distance = 3.0, -- optional
        interactDst = 1.3, -- optional
        options = {
            label = "Use ATM",
            event = "Renewed-Banking:client:openBankUI",
            atm = true
        }
    })
end

**Comment by FjamZoo** [comment]
This is more than likely due to NVE which have replaces for such models (Same reason they are hard to target with ox-target)

Try to put ignoreLos to true as that will ignore the fact you have to look at the atm

**Comment by brunoodle** [comment]
Unfortunately adding `ignoreLos = true` did not work. Reverting back to my version before the updates fixes the problem, so I don't think it's exactly an issue related to the NVE replacements

**Comment by FjamZoo** [comment]
> Unfortunately adding `ignoreLos = true` did not work. Reverting back to my version before the updates fixes the problem, so I don't think it's exactly an issue related to the NVE replacements

I added the ignoreLos = true and it works just fine.

![image](https://github.com/darktrovx/interact/assets/34508112/20fea1df-48e8-47ca-b7b5-17e9c70eac45)

**Comment by brunoodle** [comment]
Since the ignoreLos didn't work for me I tested a few different changes and got it to work. With some testing, [this was what caused the issue.](https://github.com/darktrovx/interact/commit/768a6eb333c19c898fa4da7202523d05d57347a2) I updated to the most recent version and replaced line 51 in client\interacts.lua with:

`local isClose = isPrimary and interaction.curDist <= interaction.interactDst and (interaction.entity and (interaction.entity == CurrentTarget) or true)`

I'm not sure if this is the best way or if this has any unintended side effects, but it works for me without breaking anything else.
As a side note I did have people test with/without NVE and without our server side NVE graphics and it didn't change anything.
